package.json

```diff
@@ -21,8 +21,8 @@
   "license": "MIT",
   "homepage": "https://github.com/theplatapi/react-disqus-thread",
   "peerDependencies": {
-    "react": "^15.0.1",
-    "react-dom": "^15.0.1"
+    "react": ">=15.0.1 < 17.0.0",
```

Comment:
Why not just list React 16?

Reply:
Didn't want to exclude v15. Happy to make any change you want though